CoreFoundation/Base.subproj/CFFileUtilities.c

```diff
@@ -1198,7 +1198,7 @@ static CFStringRef _CFXDGCreateHome(void) {
 }
 
 /// a single base directory relative to which user-specific data files should be written. This directory is defined by the environment variable $XDG_DATA_HOME.
-CF_SWIFT_EXPORT
```

Comment:
This should still only be exported on non-Darwin platforms. We may need another version of this `CF_EXPORT` macro.